git-link.el

```diff
@@ -432,14 +432,15 @@ return (FILENAME . REVISION) otherwise nil."
                               (format "L%s" start)))))))
 
 (defun git-link-azure (hostname dirname filename branch commit start end)
-  (format "https://%s/%s?path=%s&version=%s%s"
+  (format "https://%s/%s?path=%s&version=%s&line=%s&lineEnd=%s&lineStartColumn=1&lineEndColumn=9999&lineStyle=plain"
 	  hostname
 	  dirname
-      filename
+      (if (not (string-prefix-p "/" filename))
+          (concat "/" filename)
```

Comment:
I don't think `filename` will ever begin with a forward slash. Looking at existing handlers, they all assume it wont be there. Have you seen otherwise? If not, I think we can always include it. But note that this should be URL encoded (this may be an existing bug with other handlers) so maybe just add `path=%27%s` to the `format` template?